src/gencost/waterfall.py

```diff
@@ -1169,6 +1237,11 @@ def get_860_by_x(
                 how="left",
                 validate="m:1",
             )
+            .assign(
+                pollution_control_costs_per_kw=lambda x: x[
+                    "pollution_control_costs_per_kw"
+                ].fillna(0.0)
+            )
```

Comment:
`fillna` can be applied to a dataframe using a dict to identify the fill values for any set of columns, so this could become

```suggestion
            .fillna({"pollution_control_costs_per_kw" : 0.0})
```

It is probably marginally faster, and definitely shorter. However, the assign way is probably the better way to think about it as we look toward polars.